e2e_tests/tests/command/test_strict_ntsc.py

```diff
@@ -79,7 +79,7 @@ def assert_notebook_access(
         return
 
     assert notebook_id not in notebook_ids
-    with pytest.raises(NotFoundException):
+    with pytest.raises(NotFoundException):  # FIXME: do we want to mask that the notebook exists?
```

Comment:
if so we'd need to ensure that the master behaves the exact same way for not found and permission denied to avoid leaking this info

Reply:
we had a chat Tuesday. we do want this. I think it should be a separate PR.

Reply:
adding this in #5641